### DISCUSSION TITLE
Support for Ligatures

### DISCUSSION THREAD

**Discussion by TheBoredHuman** [fontra/fontra#1212] [General]
Hi Fontra,

I wanted to know if you had any plans to release a feature for ligatures soon? This would really help me out as I'm trying to build a font in Kannada which relies has alphasyllabary structure where the alphabet changes based on the combination of consonants and vowels.

An example of this:

**ಕ** (k) + **ಾ** (a) = **ಕಾ** (kaa represented in one glyph, but was created by typing 'k' + 'a' on the keyboard)
**ರ** (r) + **ಿ** (i) = **ರಿ** (ri represented in one glyph, but was created by typing 'r' + 'i' on the keyboard)

**Comment by justvanrossum** [comment]
Hi, thank you for your interest! 

Supporting opentype shaping (including ligatures) is definitely planned (and much desired), but exactly how soon is still to be decided. Very roughly I hope to reach this level in 3 to 6 months from now.

**Comment by justvanrossum** [comment]
I have opened https://github.com/googlefonts/fontra/issues/1211 to track progress on the more general set of features needed to support this.
